### ISSUE TITLE
Read files from system

### ISSUE THREAD

**Issue by peterzhu2118** [netblue30/firejail#911]
How can I whitelist a folder so that it reads from the outer system and not from inside the sandbox (i.e. I have a code file in the system, how can I run that in sandboxed mode)?

**Comment by netblue30** [comment]
You can use --blacklist=somefolder. The folder will be visible outside the sandbox. In the sandbox the folder will be empty, with no read access to the regular user.

**Comment by peterzhu2118** [comment]
I don't think you are understanding the question, let me rephrase it. Lets say we have a file called file.txt in location `/home/user/file.txt`. How can I make it so that that file is accessible  within firejail?

**Comment by netblue30** [comment]
By default most of the files are available in the sandbox. What file are you trying out?

**Comment by peterzhu2118** [comment]
So if I go into my /home/ folder in firejail, it it empty even through it contains a ton of stuff in the actual system.  This is the profile I am using:
```
include /etc/firejail/disable-common.inc
include /etc/firejail/disable-programs.inc
include /etc/firejail/disable-passwdmgr.inc

private
private-dev
nosound
no3d
private-tmp
blacklist /tmp/.X11-unix
seccomp
```
This is my firejail.config file (minus the comments):
```
network yes
quiet-by-default yes
```

**Comment by netblue30** [comment]
> private

In your profile file, this is the line that is creating the problem. It will mask almost all the files in your home directory. You will be left with a small skeleton:
`````
$ firejail --private
Reading profile /etc/firejail/default.profile
Reading profile /etc/firejail/disable-common.inc
Reading profile /etc/firejail/disable-programs.inc
Reading profile /etc/firejail/disable-passwdmgr.inc

** Note: you can use --noprofile to disable default.profile **

Parent pid 16287, child pid 16288
Child process initialized
[netblue@debian ~]$ ls -l
total 0
[netblue@debian ~]$ ls -al
total 28
drwx------ 3 netblue netblue   120 Nov 19 11:31 .
drwxr-xr-x 3   65534   65534    60 Nov 19 11:31 ..
-rw------- 1 netblue netblue     0 Nov 19 11:31 .asoundrc
-rw-r--r-- 1 netblue netblue  3515 Nov 19 11:31 .bashrc
drwxr-xr-x 3 netblue netblue    60 Nov 19 11:31 .config
-rw------- 1 netblue netblue 21123 Nov 19 11:31 .Xauthority
`````

You might want to look into using whitelists, where you get the same skeleton plus the files you specify in whitelist command. Look for an example in /etc/firejail/firefox.
